src/go/rpk/pkg/cli/cmd/redpanda/admin/brokers/brokers.go

```diff
@@ -161,3 +162,39 @@ the cluster leader handles the request.
 		},
 	}
 }
+
+func newDecommissionStatus(fs afero.Fs) *cobra.Command {
+	var completion int
+	return &cobra.Command{
+		Use:   "decommission-status [BROKER ID]",
+		Short: "Show the progress of a node decommissioning",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			broker, _ := strconv.Atoi(args[0])
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			cl, err := admin.NewClient(fs, cfg)
+			out.MaybeDie(err, "unable to initialize admin client: %v", err)
+
+			dbs, err := cl.DecommissionBrokerStatus(cmd.Context(), broker)
+			out.MaybeDie(err, "unable to request brokers: %v", err)
+
+			headers := []string{"Namespace-Topic", "Partition", "Moving-to", "Parttion-size", "Bytes-moved", "Bytes-remaining", "Completion (%)"}
```

Comment:
typo `parttion`

also I'd change `completion (%)` to `completion-%` -- where possible, I try to format the output for awk

Reply:
Great point.